[2024-09-26T17:01:56.749+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flats.create_fact_table manual__2024-09-26T17:01:45.298190+00:00 [queued]>
[2024-09-26T17:01:56.756+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flats.create_fact_table manual__2024-09-26T17:01:45.298190+00:00 [queued]>
[2024-09-26T17:01:56.757+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-09-26T17:01:56.773+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): create_fact_table> on 2024-09-26 17:01:45.298190+00:00
[2024-09-26T17:01:56.778+0000] {standard_task_runner.py:57} INFO - Started process 1068 to run task
[2024-09-26T17:01:56.785+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'flats', 'create_fact_table', 'manual__2024-09-26T17:01:45.298190+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/flats.py', '--cfg-path', '/tmp/tmpkklkhvjc']
[2024-09-26T17:01:56.787+0000] {standard_task_runner.py:85} INFO - Job 50: Subtask create_fact_table
[2024-09-26T17:01:56.849+0000] {task_command.py:415} INFO - Running <TaskInstance: flats.create_fact_table manual__2024-09-26T17:01:45.298190+00:00 [running]> on host 5096eb060c49
[2024-09-26T17:01:56.976+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flats' AIRFLOW_CTX_TASK_ID='create_fact_table' AIRFLOW_CTX_EXECUTION_DATE='2024-09-26T17:01:45.298190+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-26T17:01:45.298190+00:00'
[2024-09-26T17:01:56.978+0000] {sql.py:274} INFO - Executing: 
    CREATE TABLE IF NOT EXISTS fact_flat (
        metro_id SERIAL,
        text_id SERIAL,
        seller_id SERIAL,
        advt_id SERIAL,
        building_id SERIAL,
        flat_params_id SERIAL,
        district_id SERIAL,
        geo_id SERIAL,
        price DECIMAL,
        processed_dttm DATE NOT NULL,
        source VARCHAR(20) NOT NULL
    )
[2024-09-26T17:01:56.989+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-09-26T17:01:57.184+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-09-26T17:01:57.469+0000] {sql.py:418} INFO - Running statement: 
    CREATE TABLE IF NOT EXISTS fact_flat (
        metro_id SERIAL,
        text_id SERIAL,
        seller_id SERIAL,
        advt_id SERIAL,
        building_id SERIAL,
        flat_params_id SERIAL,
        district_id SERIAL,
        geo_id SERIAL,
        price DECIMAL,
        processed_dttm DATE NOT NULL,
        source VARCHAR(20) NOT NULL
    ), parameters: None
[2024-09-26T17:01:57.657+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=flats, task_id=create_fact_table, execution_date=20240926T170145, start_date=20240926T170156, end_date=20240926T170157
[2024-09-26T17:01:57.693+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-09-26T17:01:57.716+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
